src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
```

Comment:
D_on't use literals in expressions (HARDCODE). Instead use constants explaining this values_ - cheklist point, create consts for discount values